test/support/page_objects/discover_page.js

```diff
@@ -285,4 +285,55 @@ export default class DiscoverPage {
       .catch(() => false);
   }
 
+  clickFieldListItem(field) {
+    return this.findTimeout
+    .findByCssSelector('li[attr-field="' + field + '"]').click();
+  }
+
+  clickFieldListItemAdd(field) {
+    return this.findTimeout
+    .findByCssSelector('li[attr-field="' + field + '"] > div > button').click();
+  }
+
+  clickFieldListItemVisualize(field) {
+    return this.findTimeout
+    .findByCssSelector('li[attr-field="' + field + '"] > a').click();
+  }
```

Comment:
I didn't fix this.  Yes, I agree we need a lot more data-test-subjs.  I'd like to get these tests in and have another PR for adding data-test-subjs and using them in the tests.